### ISSUE TITLE
[Adaptive Cards- Import- Stock Update]: Talkback does not announces the information about the stocks going up on Stock Update pane.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7259]
https://microsoft.visualstudio.com/OS/_workitems/edit/34086428
